packages/block-editor/src/components/date-format-picker/stories/index.story.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Internal dependencies
+ */
+import DateFormatPicker from '../';
```

Comment:
It is recommended to place it after `WordPress dependencies`. The following order is used by convention in Gutenberg:

- External dependencies
- WordPress dependencies
- Internal dependencies